### ISSUE TITLE
Rebrand to 'The BED Stack'

### ISSUE THREAD

**Issue by yamcodes** [agnyz/the-bed-stack#66]
It would be really funny if we marketed this repo as 'The BED Stack' as a tongue-in-cheek to [the BETH stack](https://github.com/ethanniser/the-beth-stack).

I realized this when I worked on the docs, the tagline is literally:

> **B**un + **E**lysiaJS + **D**rizzleORM

**Comment by yamcodes** [comment]
@Hajbo @aruay99 I think it would be hilarious. What are your thoughts?

**Comment by Hajbo** [comment]
Sounds interesting! Although it'd be much more work to learn HTMX along with the rest of the current stack. I'm happy to help finish the backend but can't promise to be active on the frontend side

My only problem with turso is that it's a glorified sqlite. But definitely fits this hypebeast stack

**Comment by yamcodes** [comment]
**TL;DR**: `agnyz/bed-stack-realworld` is funnier than `agnyz/elysia-realworld-example-app`

> Sounds interesting! Although it'd be much more work to learn HTMX along with the rest of the current stack. I'm happy to help finish the backend but can't promise to be active on the frontend side
> 
> My only problem with turso is that it's a glorified sqlite. But definitely fits this hypebeast stack


You're right - learning HTMX would be extra work and it would likely be a big change as it would categorize this repo as a [fullstack RealWorld app](https://codebase.show/projects/realworld?category=fullstack) instead of a backend one. Fundamentally, our endpoints would change as instead of serving JSON they would serve HTML.

This is why, IMO, frontend is outside the scope of this repository, and if we're interested in including it and turning into fullstack we should do so in a fork or a separate repository.

On that note, at the time of writing there are already 2 HTMX implementation provided in the link above, but neither are using ElysiaJS.

This issue wasn't referring to any of that, nor Turso. I am simply suggesting to rename this project to something catchier than `Elysia RealWorld`; something that would also be indicative of the other key technologies here.

**Comment by Hajbo** [comment]
Shit I didn't read it carefully. Sure we can rename it, that shouldn't be too hard

**Comment by yamcodes** [comment]
Great! Want to hear from @aruay99 too on this one

**Comment by aruay99** [comment]
I love @yamcodes our BED so much! so I am down to have another BED we can share :joy:
